modules/rntuple.mjs

```diff
@@ -147,6 +147,24 @@ deserializeHeader(header_blob) {
 
   const fieldListCount = reader.readU32(); // number of field entries
   console.log('Field List Count:', fieldListCount);
+  for (let i = 0; i < fieldListCount; ++i) {
+    const fieldVersion = reader.readU32(),
+    typeVersion = reader.readU32(),
+    parentFieldId = reader.readU32(),
+    structRole = reader.readU8(),
```

Comment:
`structRole` and `flags` are both 16 bits (as described [here](https://github.com/root-project/root/blob/master/tree/ntuple/doc/BinaryFormatSpecification.md#field-description)).